### ISSUE TITLE
Issues installing magento2 via composer

### ISSUE THREAD

**Issue by bkeetman** [richdynamix/magestead#17]
Seems there is an issue installing magento2 via composer:

```
==> default:   - Installing magento/composer (1.0.2)
==> default:     Downloading: Connecting...
==> default:
==> default:   [Composer\Downloader\TransportException]
==> default:   The "https://repo.magento.com/archives/magento/composer/magento-composer-1.0.2.0.zip" file could not be downloaded (HTTP/1.1 404 Not Found)
==> default:
==> default:
```

This might be useful:
https://github.com/magento/magento2/issues/2419

**Comment by richdynamix** [comment]
Thanks for bringing this to my attention, I haven't seen this myself however I am in the process of doing a new version of Magestead which concentrates on using Magento 2 GA using the `Integrator, packager` installer type.

**Comment by muasir** [comment]
We have implemented few fixes in our repo regarding security.  Please follow this doc to update your credentials http://devdocs.magento.com/guides/v2.0/install-gde/prereq/connect-auth.html and try again.

**Comment by richdynamix** [comment]
Closing this ticket as issues are now resolved in Magestead 2.0. Composer is not used for installation of Magento 2.
